lib/streamlit/runtime/metrics_util.py

```diff
@@ -55,6 +55,19 @@
 
 # A list of dependencies to check for attribution
 _ATTRIBUTIONS_TO_CHECK: Final = [
+    # DB Clients:
+    "pymysql",
```

Comment:
I think `MySQLdb` is another commonly used MySQL python client that we might want to track metrics for

Reply:
Added 👍